README.md

```diff
@@ -1,7 +1,7 @@
 Grofer
 ======
 
-[![pesos](https://circleci.com/gh/pesos/grofer.svg?style=svg)](https://app.circleci.com/pipelines/github/pesos/grofer)
+[![pesos](https://circleci.com/gh/pesos/grofer.svg?style=svg)](https://app.circleci.com/pipelines/github/pesos/grofer) [![Docker](https://github.com/pesos/grofer/actions/workflows/docker-publish.yml/badge.svg)](https://github.com/pesos/grofer/actions/workflows/docker-publish.yml) [![Go](https://github.com/pesos/grofer/actions/workflows/go.yml/badge.svg)](https://github.com/pesos/grofer/actions/workflows/go.yml)
```

Comment:
I think the badges are changed in #132 too. Is there a specific reason for them to be here?

Reply:
Ah these were the older workflows. I'll remove them here as the other PR (#132) will have all of the updated badges in a uniform manner.